virt/virt-4-13-release-notes.adoc

```diff
@@ -94,7 +94,7 @@ The SVVP Certification applies to:
 //CNV-25427 Release note: NEW Allow VM admin to expose SSH service using LB in the UI
 
 //CNV-25574 Release note: NEW UI - Ability to make a hot-plug disk part of the VM
-
+* The option to make a hot-plug volume a persistent volume has been added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].
```

Comment:
```suggestion
* The option to make a hot-plug volume a persistent volume is added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].
```
Use simple present tense in most cases. See https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense.

Reply:
Fixed. Thanks!